app/services/hyrax/thumbnail_path_service.rb

```diff
@@ -5,7 +5,7 @@ class << self
       # @param [#id] object - to get the thumbnail for
       # @return [String] a path to the thumbnail
       def call(object)
-        return default_image unless object.try(:thumbnail_id)
+        return default_image unless object.try(:thumbnail_id)&.to_s&.present?
```

Comment:
For collections shared spec, the thumbnail_id was a Valkyrie ID with id="".  I don't know if this would happen in the application, but it doesn't hurt anything to have the extra code.  If thumbnail_id is nil, it will still use the default_image.

Reply:
This may be a separate commit. There may be other problems involved. @no-reply can you take action on this?

Reply:
We may need to test that Valkyrie::ID with an empty string is not present.